src/app/components/progress-indicator/index.html

```diff
@@ -103,31 +103,31 @@
     <p>
       The progress indicator nav button component allows users to navigate the steps in modal wizards. We recommend against using it in waterfall pages and passive progress indicators.
     </p>
+    <sky-demo-page-property
+      propertyName="progressIndicator"
+    >
+      Specifies the progress indicator component to associate with the nav button. This property accepts an array or Observable array of <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="buttonText"
       isOptional="true"
       defaultValue="Next"
     >
-      Specifies the label to display on the nav button.
+      Specifies the label to display on the nav button. The property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="buttonType"
       isOptional="true"
       defaultValue="next"
     >
-      Specifies the type of nav button to include. Options include <stache-code>finish</stache-code>, <stache-code>next</stache-code>, and <stache-code>previous</stache-code>.
+      Specifies the type of nav button to include. Options include <stache-code>finish</stache-code>, <stache-code>next</stache-code>, and <stache-code>previous</stache-code>. This property accepts <stache-code>string</stache-code> values.
```

Comment:
For consistency with other descriptions where we list options, I'd change "Options include" to "The valid options are"

Reply:
✔️